.azure/pipelines/pr/Phonebook.Source.MockPeopleBackend.pr.yml

```diff
@@ -0,0 +1,18 @@
+trigger: none
```

Comment:
maybe we should trigger a build for the mock backend to check if it's healthy.

Reply:
There is indeed a PR Trigger for the PR right beneath: https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml#pr-triggers